recipes/BinauralWSJ0Mix/separation/train.py

```diff
@@ -682,13 +709,30 @@ def audio_pipeline_s3(s3_wav):
         )
```

Comment:
![image](https://user-images.githubusercontent.com/73161104/183464526-d6a4ab6b-a0c8-479d-bac1-be693ed99019.png)
Hi @ycemsubakan, I don't know how to check the scripts. The simliest way is to delete this 'if' command. But I think the dataset generation scripts won't change once this PR merged so we don't need to correct it.